### PR TITLE
Fix: Move Hamburger Button to the Left

### PR DESCRIPTION
**Pull request by camitaylor**
# [Move Hamburger Button to the Left](hhttps://trello.com/c/Dqrih04U/44-move-hamburger-button-to-the-left)

## A short description of what this PR does.
As a user and developer,  I would like the hamburger menu button to be on the left side for aesthetic purposes. 

## UI
Screen Shot 2021-09-16 at 10 37 52 AM

## How was it tested?
This was linted with Yarn and tested with Jest.

Screen Shot 2021-09-16 at 9 56 37 AM

### Yarn Lint
Screen Shot 2021-09-16 at 9 57 06 AM

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/aaronshivers/snapit-hacktoberfest/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

### CODE REVIEW

File: src/components/Navigation/Navigation.jsx [hunk 1]
```diff
@@ -95,11 +98,36 @@ const Navigation = () => {
   const anchor = isDesktopOrLaptop ? "left" : "top";
 
   return (
-    <div data-test="navigation-display">
+    <div data-test="navigation-display" style={{ marginBottom: "125px" }}>
```
Comment (aaronshivers) [inline src/components/Navigation/Navigation.jsx:101]: 🔧 Let's use mui styling as much as possible.
https://mui.com/system/spacing/

File: src/components/Navigation/Navigation.jsx [hunk 2]
```diff
@@ -95,11 +98,36 @@ const Navigation = () => {
...
+                </IconButton>
+              </Grid>
+              <Grid item xs={4} />
+            </Grid>
+            <Grid xs={2} item>
+              <Grid container justify="space-between">
+                <img
+                  src="../../src/assets/SnapIT-logo.png"
+                  alt="logo"
+                  style={{ height: "40px", marginLeft: "50px" }}
```
Comment (aaronshivers) [inline src/components/Navigation/Navigation.jsx:125]: 🔧 Same here, and anywhere else, let's use mui styling properties.

File: src/components/Navigation/Navigation.jsx [hunk 3]
```diff
@@ -95,11 +98,36 @@ const Navigation = () => {
...
+                  data-test="nav-button"
+                  onClick={toggleDrawer(anchor, true)}
+                >
+                  <MenuIcon />
+                </IconButton>
+              </Grid>
+              <Grid item xs={4} />
+            </Grid>
+            <Grid xs={2} item>
+              <Grid container justify="space-between">
```
Comment (aaronshivers) [inline src/components/Navigation/Navigation.jsx:121]: https://mui.com/system/grid/

File: src/components/Navigation/Navigation.test.jsx [hunk 4]
```diff
@@ -28,6 +28,12 @@ describe("Navigation", () => {
     expect(navigationDisplay.length).toBe(1);
   });
 
+  test("renders AppBar display", () => {
+    const wrapper = setup();
+    const appBarDisplay = findByTestAttribute(wrapper, "app-bar-display");
+    expect(appBarDisplay.length).toBe(1);
```
Comment (aaronshivers) [inline src/components/Navigation/Navigation.test.jsx:34]: ❓ Can you try `expect(appBarDisplay.exists()).toBeTruthy();`?

**Review by aaronshivers — CHANGES_REQUESTED** [review]

File: src/components/Navigation/Navigation.jsx (hunk 1 above)
Comment (camitaylor) [reply src/components/Navigation/Navigation.jsx]: Addressed: commit d432723ed287368994edd0de8aae243c7d9ed62e

File: src/components/Navigation/Navigation.jsx (hunk 2 above)
Comment (camitaylor) [reply src/components/Navigation/Navigation.jsx]: Addressed: commit d432723ed287368994edd0de8aae243c7d9ed62e

File: src/components/Navigation/Navigation.test.jsx (hunk 4 above)
Comment (camitaylor) [reply src/components/Navigation/Navigation.test.jsx]: Addressed: commit d432723ed287368994edd0de8aae243c7d9ed62e

File: src/components/Navigation/Navigation.jsx (hunk 3 above)
Comment (camitaylor) [reply src/components/Navigation/Navigation.jsx]: Addressed: commit d432723ed287368994edd0de8aae243c7d9ed62e

File: src/components/Navigation/Navigation.jsx [hunk 5]
```diff
@@ -82,9 +86,37 @@ const Navigation = () => {
...
+                  >
+                    <MenuIcon />
+                  </IconButton>
+                </Grid>
+                <Grid item xs={9} />
+                <Grid item xs={2}>
+                  <img
+                    src="../../src/assets/SnapIT-logo.png"
+                    alt="logo"
+                    style={{ height: 40 }}
```
Comment (aaronshivers) [inline src/components/Navigation/Navigation.jsx:113]: ⛏️  It looks like they want us to use `sx` now.
https://mui.com/system/the-sx-prop/

```suggestion
                    sx={{ height: 40 }}
```

Comment (camitaylor) [reply src/components/Navigation/Navigation.jsx]: Addressed: commit d6259ad3db873d2a960329c601ca0120b6ff4ee6

**Review by aaronshivers — APPROVED** [review]

**Pull request merged**